requirements.txt

```diff
@@ -10,3 +10,5 @@ oauth2client>=4.1.3
 python-openstackclient
 gitpython
 paramiko
+openshift-client
```

Comment:
Did you want to pin a version for the oc client and python-ipmi?

Reply:
I checked and those parts of the API appear static and included in all versions, so I don't think that's necessary unless you think it's a good idea to prevent future breaking changes.